PSScriptAnalyzerSettings.psd1

```diff
@@ -6,5 +6,6 @@
         'PSUseShouldProcessForStateChangingFunctions'
         'PSUseApprovedVerbs'
         'Measure-SafeCommands'
+        'PSAvoidUsingWriteHost'
```

Comment:
Wouldn't this raise a lot of PSScriptAnalyzer warnings all over the codebase? We couldn't simply replace `Write-Host` with `Write-Output` given there is no `-BackgroundColor/-ForegroundColor/-NoNewLine` options for that cmdlet. Probably best doing this work in another issue anyways, doesn't seem to belong here in my opinion.

Reply:
I guess that's why it's set under excludedrules. 🙂

Reply:
Oops, I guess thats what I get for looking at code late in the night 😄

Reply:
It would be great if someone could put in a PR to turn off that rule in PSScriptAnalyzer by default.